src/Views/AppInfoView.vala

```diff
@@ -540,5 +552,34 @@ namespace AppCenter.Views {
                 });
             }
         }
+
+        class FundButton : FlatButton {
+            private Widgets.HumblePopover selection;
+
+            public FundButton (AppCenterCore.Package package) {
+                base (_("Fund"), "credit-card-symbolic");
+
+                selection = new Widgets.HumblePopover (this, true);
+                selection.payment_requested.connect ((amount) => {
+                    var stripe = new Widgets.StripeDialog (amount,
+                                                           package.get_name (),
+                                                           package.component.get_desktop_id ().replace (".desktop", ""),
+                                                           package.get_payments_key ()
+                                                          );
+
+                    stripe.download_requested.connect (() => {
+                        Settings.get_default ().add_paid_app (package.component.get_id ());
+                    });
+
+                    stripe.show ();
+                });
+
+                tooltip_text = _("Fund the development of this project");
```

Comment:
I think we should use "app" here instead of "project". I feel like "project" carries hobby connotations and sounds less like a real valuable product